pinax/stripe/hooks.py

```diff
@@ -51,7 +51,7 @@ def send_receipt(self, charge, email=None):
                 to=[email],
                 from_email=settings.PINAX_STRIPE_INVOICE_FROM_EMAIL
             ).send()
-            charge.receipt_sent = num_sent > 0
+            charge.receipt_sent = num_sent and num_sent > 0
```

Comment:
You could do that also `bool(num_sent) > 0`.
reads better I think.

Reply:
Not motivated enough to change that now.

Reply:
fair enough